packages/components/organisms/f-checkout/src/components/Checkout.vue

```diff
@@ -138,10 +129,17 @@ import {
     VUEX_CHECKOUT_MODULE
 } from '../constants';
 import checkoutValidationsMixin from '../mixins/validations.mixin';
+import loggerMixin from '../mixins/logger.mixin';
 import EventNames from '../event-names';
 import tenantConfigs from '../tenants';
 import { mapUpdateCheckoutRequest, mapAnalyticsNames } from '../services/mapper';
 
+const {
+    CreateGuestUserError,
+    UpdateCheckoutError,
+    PlaceOrderError
+} = exceptions;
```

Comment:
If you wanted, you could move this to the import: `import { ..., ..., ... } from '...';`

Reply:
I did but it complained that it could not find the Error classes so @benwhite-deltas suggested to do it this way, which worked.